src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -298,4 +315,47 @@ private HapiOperationOutcome handleJsonProcessingException() {
                 + "but outcome not able to be interpreted!")
         .build();
   }
+
+  protected String enforcePatientId(String testCaseJson, TestCase testCase) {
```

Comment:
Is there a difference between testCaseJson and testCase.getJson() ? also any reason we need this method needs to be protected instead of private ?

Reply:
@RohitKandimalla I should be able to just use testCase.getJson(). As to the protected method, it's much easier for testing.

Reply:
@RohitKandimalla Changed the parameters into one. Thanks!